src/node.rs

```diff
@@ -88,16 +111,18 @@ impl Sprawl {
         Ok(node)
     }
 
-    /// Removes all nodes.
+    /// Removes all nodes
     ///
-    /// All associated nodes will be invalid.
+    /// All associated [`Id`] will be rendered invalid.
     pub fn clear(&mut self) {
         self.nodes_to_ids.clear();
         self.ids_to_nodes.clear();
         self.forest.clear();
     }
 
-    /// Remove nodes.
+    /// Remove a specific [`Node`] from the tree
+    ///
+    /// Its [`Id`] is rendered invalid
```

Comment:
This is minor, but maybe "rendered" isn't the best choice of words here when we expect this crate to be used in a game dev context.
```suggestion
    /// Its [`Id`] is marked as invalid.
```